### PR TITLE
 External LB image update to use latest f5 vendor dependency

### PR DESCRIPTION
**Pull request by alena1108**
https://github.com/rancher/rancher/issues/13269

### CODE REVIEW

File: templates/f5/1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+external-lb:
+  image: rancher/external-lb:v0.3.6
+  command: -provider=f5_BigIP
+  expose: 
+   - 1000
+  environment:
+    F5_BIGIP_HOST: ${F5_BIGIP_HOST}
+    F5_BIGIP_USER: ${F5_BIGIP_USER}
+    F5_BIGIP_PWD: ${F5_BIGIP_PWD}
+    LB_TARGET_RANCHER_SUFFIX: ${LB_TARGET_RANCHER_SUFFIX}
```
Comment (rawmind0) [inline templates/f5/1/docker-compose.yml]: Getting error message `level=fatal msg="Failed to initialize Rancher API client: Environment variable 'CATTLE_URL' is not set"`

Could you please add label to get `CATTLE_URL` initialized??

```
+  labels:
+    io.rancher.container.create_agent: "true"
+    io.rancher.container.agent.role: "external-dns"
```

File: templates/f5/1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    CATTLE_URL: ${CATTLE_URL}
+    CATTLE_ACCESS_KEY: ${CATTLE_ACCESS_KEY}
+    CATTLE_SECRET_KEY: ${CATTLE_SECRET_KEY}
```
Comment (rawmind0) [inline templates/f5/1/docker-compose.yml]: These env variables ares added automatically with labels, don't need to add them manually, could you please remove them?

```
-    CATTLE_URL: ${CATTLE_URL}
-    CATTLE_ACCESS_KEY: ${CATTLE_ACCESS_KEY}
-    CATTLE_SECRET_KEY: ${CATTLE_SECRET_KEY}
```

Comment (alena1108) [reply templates/f5/1/docker-compose.yml]: updated

**Comment by rawmind0** [comment]
Tested and working... LGTM

**Comment by patrick0057** [comment]
The fix has been tested and confirmed working.

**Comment by patrick0057** [comment]
Unfortunately it appears that the ipv4 and ipv6 mismatch errors are still showing up:

1/21/2019 11:47:26 AMtime="2019-01-21T17:47:26Z" level=error msg="Failed to add LB config for endpoint REDACTED: 0107028c:3: The source (::) and destination (10.0.0.4) addresses for virtual server (REDACTED) must be be the same type (IPv4 or IPv6)."

**Pull request merged**